proto/manifest.proto

```diff
@@ -12,22 +12,26 @@ message Resource {
     // Path specifies the path from the bundle root. If more than one
     // path is present, the entry may represent a hardlink, rather than using
     // a link target. The path format is operating system specific.
+    // Path MAY be an absolute path, but it MUST be interpreted with the bundle root.
```

Comment:
current `manifest_test.go` uses relative path but `continuity build` uses absolute path.
Suggestion: can we prohibit absolute path for ease of robust implementation?